src/main/kotlin/app/batch/DirectoryWriter.kt

```diff
@@ -103,7 +108,7 @@ class DirectoryWriter(private val keyService: KeyService,
     private lateinit var tableName: String
 
     @Value("\${compress.output:false}")
-    private var compressOutput: Boolean = false
+    private var compressOutput: Boolean = true
```

Comment:
Doesn't the @Value annotation contain a default of false but this will override it to true?